moveit_planners/ompl/ompl_interface/scripts/generate_state_database.cpp

```diff
@@ -54,8 +54,8 @@ static const std::string ROBOT_DESCRIPTION = "robot_description";
 
 static const std::string CONSTRAINT_PARAMETER = "constraints";
 
-static bool getUintParameterOr(const rclcpp::Node::SharedPtr& node, const std::string& param_name,
-                               size_t&& result_value, const size_t default_value)
+static bool get_uint_parameter_or(const rclcpp::Node::SharedPtr& node, const std::string& param_name,
```

Comment:
You need to keep the original function name - due to imposed naming conventions in this repo.
```suggestion
static bool getUintParameterOr(const rclcpp::Node::SharedPtr& node, const std::string& param_name,
```